docs/DeveloperGuide.adoc

```diff
@@ -279,13 +279,22 @@ Refer to the guide <<DevOps#, here>>.
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing over mouse input
-* is reasonably comfortable using CLI apps
+* Undergraduate CS student studying in NUS
+* Needs to manage his/her study plans and module planning
+* Needs to see whether his/her study plans are feasible 
+* Prefer desktop apps over other types
+* Can type fast
+* Prefers typing over mouse input
+* Is reasonably comfortable using CLI apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*:
+
+* Functionality is not offered by any other existing application.
+* Tailored to needs of NUS CS undergraduate students.
+* Users will be able to check all the problems (e.g. graduation requirements, prerequisite for modules) with their current study plan with ModBuddy through a desktop application.
+* Have multiple study plans, and move semesters around quickly with a CLI.
+* Version control for saving history of study plans, ensuring that mistakes or past study plans are recoverable.
+* Suits users who are able to type fast and can manage their study plan faster.
 
 [appendix]
 == User Stories
```

Comment:
Is the user story format correct? Perhaps you can fill in the "So that I can" column?

Reply:
Will a product survey comparing to NUSmods be helpful for your product?

Reply:
On the whole, quite a comprehensive collection of user stories! Good job!!